src/plugins/visualizations/public/wizard/group_selection/group_selection.tsx

```diff
@@ -59,7 +93,7 @@ function GroupSelection(props: GroupSelectionProps) {
           ...props.visTypesRegistry.getAliases(),
           ...props.visTypesRegistry.getByGroup(VisGroups.PROMOTED),
           // Include so TSVB still gets displayed
-          ...props.visTypesRegistry.getByGroup(VisGroups.LEGACY),
+          // ...props.visTypesRegistry.getByGroup(VisGroups.LEGACY),
```

Comment:
can we remove this line?

Reply:
https://github.com/elastic/kibana/pull/195966/commits/17197c21d02b8bbc8e86c8e5135ed8258d4fb177